docs/content/en/docs/reference/metrics.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "Metrics Reference"
+description: >
+  This reference documents Prometheus metrics exposed by Tetragon.
+weight: 4
+---
+<!-- This file is autogenerated via https://github.com/isovalent/metricstool -->
```

Comment:
```suggestion
{{< comment >}}
This page is autogenerated via https://github.com/isovalent/metricstool,
please do not edit directly.
{{< /comment >}}
```
Please use the `comment` shortcode so that Hugo removes this before generating the website, with HTML comments, they end up in the final file served by the HTTP server and are still here in prod.

Reply:
See
<img width="1390" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/078197da-ab65-4daf-9218-69004f87b8ad">

Reply:
I opted to drop this comment completely and document that the page is autogenerated in the frontmatter description instead. Do you think that's enough?

Reply:
ok, done